kuka_kr5_support/config/kuka_kr5.rviz

```diff
@@ -0,0 +1,250 @@
+Panels:
```

Comment:
Probably can also be reused for all robots. We should probably rename this to `view_robot` or just keep the other file.

Or is something not working in that case?

Reply:
both same, leaving view_robot in place and removing other duplicate

Reply:
will move it to kuka_resources then as it can be reused by all others